README.md

```diff
@@ -541,6 +541,53 @@ For example, using the standard location of the default user profile on macOS:
 }
 ```
 
+### Performance traces
+
+Once you determine that something is slower or faster in comparison to something
+else, investigating why is natural next step. To assist in determining why,
+consider collecting performance traces. These traces can be used to determine
+what the browser is doing differently between two version of code.
+
+When the `trace` option is turned on in Chromium-based browsers, each tachometer
+sample will produce a JSON file that can be viewed in Chromium's `about:tracing`
+tool. Enter `about:tracing` in the URL bar of Chromium, click load, and select
+the `json` file you want to view. Check out the [about:tracing doc
+page](https://www.chromium.org/developers/how-tos/trace-event-profiling-tool) to
+learn more about using the trace event profiling tool.
+
+To turn on tracing with the default configuration, add `trace: true` to a
+Chromium browser's config object. This config turns on tracing with some
+[default categories enabled](./src/defaults.ts#L29) and puts the JSON files into
```

Comment:
Would be nice to also have a link about other available categories. I couldn't find anything great from a Google search though. Do you happen to know of one? There's a full list at https://chromium.googlesource.com/chromium/src/+/master/base/trace_event/builtin_categories.h, but no explanations...

Reply:
Hmm I'm not aware of one either. I added a little section below describing how to find the available categories from a browser. Again, no explanations there either but at least it is a little more approachable than reading C++ 😅